.travis.yml

```diff
@@ -44,6 +44,25 @@ jobs:
       python: '3.7'
       sudo: required
       dist: xenial
+    - &test-macos
+      language: generic
+      os: osx
+      osx_image: xcode9.4
+      sudo: required
+      env: TOXENV=py27
```

Comment:
Should not be defined in the "included" part probably ("&test-macos").

Reply:
done

Reply:
It's good to have a default which get changed later.